app/views/partials/_navbar.html.slim

```diff
@@ -0,0 +1,27 @@
+header
+  nav.navbar.navbar-expand-md.navbar-dark.fixed-top.bg-dark
+    img[src="/proconf_logo.jpg" alt="proConf logo" height="30" width="30"]
```

Comment:
We really need this logo 2 times?
Same as for prev

Reply:
Yes, it's different places